data/Airmail.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
This file name should be lowercase so "airmail.json".

Reply:
Same for the image file. the name should be the same as the identifier, so "airmail.png"